lib/debug.py

```diff
@@ -140,11 +145,45 @@ def init_debugger(self):
         #    PyDev debugger.)" statement, below. Uncheck the box to have it continue
         #    automatically to your first set breakpoint.
 
+        ###
+
+        # To debug with VS Code
+        # see: https://code.visualstudio.com/docs/python/debugging#_command-line-debugging
+        #      https://code.visualstudio.com/docs/python/debugging#_debugging-by-attaching-over-a-network-connection
+        # 
+        # 1. Install the Python extension for VS Code
+        #      pip install debugpy
+        # 2. create .vscode/launch.json containing somewhere:
+        #       "configurations": [ ...
+        #           {
+        #               "name": "Python: Attach",
+        #               "type": "python",
+        #               "request": "attach",
+        #               "connect": {
+        #                 "host": "localhost",
+        #                 "port": 5678
+        #               }
+        #           }
+        #       ]
+        # 3. Touch a file named "DEBUG" at the top of your git repo, as above.
+        #     containing "vscode" or "vscode-script" see parse_file() in debug_mode.py for details
+        # 4. Start the debug server in VS Code by clicking on the debug icon in the left pane
+        #    select "Python: Attach" from the dropdown menu and click on the green arrow
+        #
+        # Notes:
+        #   to see flask server url routes:
+        #      - comment out the line self.disable_logging() in run() of lib/api/server.py
+                
+
         try:
-            if 'PYCHARM_REMOTE_DEBUG' in os.environ:
```

Comment:
Thanks, this turned out to be kinda clunky.  I like the idea of putting the debugger type in the DEBUG file.